sakai-evaluation-tool/src/java/org/sakaiproject/evaluation/tool/utils/RenderingUtils.java

```diff
@@ -376,7 +379,7 @@ public static void renderResultsColumn(UIBranchContainer container, EvalEvaluati
             // indicate the viewable date as well
             resultOutput.decorate(new UITooltipDecorator(
                     UIMessage.make("controlevaluations.eval.report.viewable.on", new Object[] { viewableDate }) ));
-        } else if (!evalResultsViewable) {
+        } else if (now.before(viewDate) || !evalResultsViewable) {
```

Comment:
Looks like it :
https://github.com/hec-montreal/evaluation/blob/a6681dd9d1e60ab5abaf973269ea37a3a2886276/sakai-evaluation-tool/src/java/org/sakaiproject/evaluation/tool/utils/RenderingUtils.java#L349